package.json

```diff
@@ -24,26 +24,26 @@
   "author": "Felix Kling",
   "license": "MIT",
   "dependencies": {
-    "@babel/core": "^7.23.0",
-    "@babel/parser": "^7.23.0",
-    "@babel/plugin-transform-class-properties": "^7.22.5",
-    "@babel/plugin-transform-modules-commonjs": "^7.23.0",
-    "@babel/plugin-transform-nullish-coalescing-operator": "^7.22.11",
-    "@babel/plugin-transform-optional-chaining": "^7.23.0",
-    "@babel/plugin-transform-private-methods": "^7.22.5",
-    "@babel/preset-flow": "^7.22.15",
-    "@babel/preset-typescript": "^7.23.0",
-    "@babel/register": "^7.22.15",
-    "babel-core": "^7.0.0-bridge.0",
+    "@babel/core": "^7.24.7",
+    "@babel/parser": "^7.24.7",
+    "@babel/plugin-transform-class-properties": "^7.24.7",
+    "@babel/plugin-transform-modules-commonjs": "^7.24.7",
+    "@babel/plugin-transform-nullish-coalescing-operator": "^7.24.7",
+    "@babel/plugin-transform-optional-chaining": "^7.24.7",
+    "@babel/plugin-transform-private-methods": "^7.24.7",
+    "@babel/preset-flow": "^7.24.7",
+    "@babel/preset-typescript": "^7.24.7",
+    "@babel/register": "^7.24.6",
+    "babel-core": "^6.26.3",
```

Comment:
why does it depend on both `@babel/core` 7 and `babel-core` 6 ?

Reply:
@stof This was an oversight and it's already been fixed.